### PR TITLE
Practice Ammunition Tech

### PR DESCRIPTION
**Pull request by Nairodian**
## About the PR
Adds a practice ammunition technology and recipes for science to unlock.
Tier 1 for 2000 points.
Includes all the basic types of rounds in practice form but also practice disablers and laser rifles.

Reorders the dynamicRecipes list to be alphabetical as requested.
## Why / Balance
A very niche tech, but it's good for people who want to create ranges or actually use practice gear for teachings. There wasn't a way to get any of these before unless mapped so that handles that as well.

## Media
image
- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl:
- add: Science can now research practice ammunition for security.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -554,6 +554,13 @@
...
+      - ShellShotgunPractice
+      - MagazineBoxPistolPractice
+      - MagazineBoxMagnumPractice
+      - MagazineBoxLightRiflePractice
+      - MagazineBoxRiflePractice
+      - WeaponLaserCarbinePractice
+      - WeaponDisablerPractice
       - Signaller
       - SignalTrigger
       - VoiceTrigger
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:566]: Can you alphabetise all these recipes please?

**Comment by LankLTE** [comment]
I honestly thought these were roundstart in the seclathe. I’m fine with them being a research though.

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml (hunk 1 above)
Comment (Nairodian) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:566]: How so exactly? Just the practice section or for the entire list?

Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:566]: Entire list is preferable as people will just copy whatever formatting is there most of the time.

Comment (Nairodian) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml]: Fair enough, fixed!

**Comment by LankLTE** [comment]
Oh wait this wasn't merged yet

**Review by LankLTE — APPROVED** [review]

**Pull request merged**